README.md

```diff
@@ -38,7 +38,7 @@ In top level build file
 
 ```kotlin
 plugins {
-     id("org.jetbrains.kotlinx.kover") version "0.4.1"
+     id("org.jetbrains.kotlinx.kover") version "0.4.2"
```

Comment:
Please don't forget to have a changelog in the form of `CHANGES.md` and GH release

Reply:
OK, I will create them in a separate commit because I want to add a few more fixes to the release.